internal/store/serverless.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package store
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/mongodb/mongocli/internal/config"
+	atlas "go.mongodb.org/atlas/mongodbatlas"
+)
+
+//go:generate mockgen -destination=../mocks/mock_serverless.go -package=mocks github.com/mongodb/mongocli/internal/store ServerlessLister
+
+type ServerlessLister interface {
+	ListServerlessClusters(string, *atlas.ListOptions) (*atlas.ClustersResponse, error)
+}
+
+func (s *Store) ListServerlessClusters(projectID string, listOps *atlas.ListOptions) (*atlas.ClustersResponse, error) {
+	switch s.service {
+	case config.CloudService:
+		result, _, err := s.client.(*atlas.Client).ServerlessInstances.List(context.Background(), projectID, listOps)
```

Comment:
The name of this file should be based on the service that is calling `ServerlessInstances`